ingest_cleanup.py

```diff
@@ -0,0 +1,36 @@
+import concurrent.futures
+import boto3
+
+if __name__ == '__main__':
+    client = boto3.client('stepfunctions')
+
+    tasks_to_kill = []
+    args = {
+        'stateMachineArn': 'arn:aws:states:us-west-2:322322076095:stateMachine:sharedsbx-IngestGranule',
```

Comment:
Is this a hard coded value?

Reply:
It is. This was just a quick implementation to quickly kill all of the ingest workflows this can spawn.